docs/src/GettingStartedPage.js

```diff
@@ -21,7 +21,12 @@ const Page = React.createClass({
                   <h2 id="setup" className="page-header">Setup</h2>
                   <p className="lead">You can import the lib as AMD modules, CommonJS modules, or as a global JS script.</p>
 
-                  <p>First add the Bootstrap CSS to your project; check <a href="http://getbootstrap.com/getting-started/" name="Bootstrap Docs">here</a> if you have not already done that. Then:</p>
+                  <p>First add the Bootstrap CSS to your project; check <a href="http://getbootstrap.com/getting-started/" name="Bootstrap Docs">here</a> if you have not already done that. Note that:</p>
+                  <ul>
+                    <li>Because many folks use custom bootstrap themes, we do not directly depend on Bootstrap. It is up to you to to determine how you get and link to the bootstrap css and fonts.</li>
+                    <li>React-Bootstrap doesn't really depend on any bootstrap version (unless there are big breaking changes in the main lib, which are pretty rare). Just have bootstrap in your dependencies and ensure your build can read your less/sass/scss entry point.</li>
```

Comment:
This isn't necessarily true, we have built this library with Bootstrap v3.x in mind. I've had problems getting it to work with a legacy site we are migrating at work that was built against Bootstrap 2.x. It would probably be best to pull the current version of bootstrap we are running against and note that that is the version these docs are using. That way the version would stay dynamic with when we update our bootstrap dev dependency. It may be possible that this library will work with versions of bootstrap that are not too distant from the current one.